client-course-schedulizer/src/components/Tabs/FacultySchedule/FacultySchedule.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Calendar } from "../../reuseables/Calendar";
+import { ScheduleToolbar } from "../../Toolbar/ScheduleToolbar";
+import "./FacultySchedule.scss";
+
+// TODO: remove this
+const professors = ["Norman", "Adams", "VanderLinden", "Arnold", "Bailey"];
+
+export const FacultySchedule = () => {
+  return (
+    <>
+      <ScheduleToolbar />
+      <div className="calendars-wrapper">
+        <div className="calendars-adjacent">
+          {professors.map((prof, index) => {
+            return (
+              <div key={prof} className={`calendar ${index !== 0 ? "hide-axis" : ""}`}>
```

Comment:
This is good, but it makes the first professor's calendar a bit smaller than the others

Reply:
Yeah, I'm not sure the best way to fix this. I'll leave a TODO and recan revisit this later.